chromium_src/chrome/browser/ui/views/frame/browser_root_view.cc

```diff
@@ -7,14 +7,15 @@
 
 #include "brave/browser/ui/views/tabs/vertical_tab_utils.h"
 
-#define ConvertPointToTarget(THIS, TARGET_GETTER, POINT)                  \
-  if (views::View* target_v = TARGET_GETTER;                              \
-      tabs::utils::ShouldShowVerticalTabs(browser_view_->browser()) &&    \
-      (target_v == tabstrip() || !THIS->Contains(target_v))) {            \
-    ConvertPointToScreen(target_v, POINT);                                \
-    ConvertPointFromScreen(THIS, POINT);                                  \
-  } else {                                                                \
-    ConvertPointToTarget(THIS, target_v, POINT);                          \
+// Workaround for vertical tabs to work with drag&drop of text/links.
+#define ConvertPointToTarget(THIS, TARGET_GETTER, POINT)               \
```

Comment:
Oh, nice fix!
As this `POINT` is from `THIS`, we should call `ConvertPointToScreen()` with `THIS` and `POINT`. 👍🏼